cmake/kokkos_arch.cmake

```diff
@@ -956,6 +959,11 @@ ENDIF()
 
 FOREACH(ARCH IN LISTS SUPPORTED_AMD_ARCHS)
   IF (KOKKOS_ARCH_${ARCH})
+    STRING(REGEX MATCH "942" IS_942 ${ARCH})
+    IF(IS_90A)
```

Comment:
```suggestion
    IF(IS_942)
```

Reply:
Ah, my bad, messed this up in the rebase -- good catch!